Documentation/contributing/release/backports.rst

```diff
@@ -178,6 +178,15 @@ Creating the Backports Branch
    backport reviewers towards which changes may deviate from the original
    commits to ensure that the changes are correctly backported.
 
```

Comment:
We should also document that backporters should look for current outstanding backports (https://github.com/cilium/cilium/pulls?q=is%3Aopen+is%3Apr+label%3Akind%2Fbackports) and reach out to the previous backporter (either DM or in #launchpad) to discuss those backports and how they can help get them in.